core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/async/ChecksumCalculatingAsyncRequestBody.java

```diff
@@ -41,6 +42,8 @@
 @SdkInternalApi
 public class ChecksumCalculatingAsyncRequestBody implements AsyncRequestBody {
 
+    public static final int DEFAULT_CHUNK_SIZE = 16 * 1024;
+
```

Comment:
for Sync DEFAULT_CHUNK_SIZE in AwsChunkedEncodingInputStream was 128 * 1024 from begining.

For Async since we started by taking Async File Request body as a reference , I took the default chunk size as specified for AsyncFileRequestBody.